posts/2015-04-10-symfony-live-2015.md

```diff
@@ -0,0 +1,214 @@
+* SonataRestAdminBundle : administrer des contenus depuis plusieurs sources via APIs. Devrait être open-sourcé sous peu.
+* Nginx pour gérer le crop/resize des images afin de ne pas stocker les images temporaires sur le filesystem
+* Création d'un [client pour consommer une API Hateoas (HAL)](https://github.com/ekino/php-hal-client)
+
+
+## ElasticSearch dans une infrastructure Symfony2 par Nicolas Badey
+
+Présentation d'ElasticSearch avec Elastica et son intégration dans Symfony2.
+
+[Speakerdeck de la présentation ElasticSearch](https://speakerdeck.com/nicolasbadey/elasticsearch-with-elastica-in-symfony2-architecture)
+
+
+## OpenClassrooms - Le pattern View Model avec Symfony2 par Kuzniak Romain
+
+Le pattern View Model est un pattern simple qui permet d’apporter des solutions à beaucoup de problèmes :
+* Découpler la logique métier de la présentation
+* Permettre le refactoring
+* Faciliter les tests
+* Permettre de paralléliser le travail front et back
+* Construire des ressources api
+* Contrat entre le front et le back sur le contenu d'un ViewModel
+
+Superbe et simple introduction à ce pattern. En savoir plus : [Pattern Modèle-Vue-VueModèle](http://fr.wikipedia.org/wiki/Mod%C3%A8le-Vue-VueMod%C3%A8le)
+
+
+## Meetic backend mutation with Symfony par Joris Calabrese
+
+Comment Meetic opère son changement technologique sur son SI. De la création d’API jusqu’à la mise en place d’une démarche qualité tout en passant par l'adoption du Behavior Driven Development.
+
+En vrac :
+* Migration d'un code monolithique vers des API REST en Sf2
+* Exemple de microservices : AB Test, GEO, Permission, Configuration
+* Déploiement avec Composer, Satis, Sf2 et Capistrano sur des centaines de serveurs
+* Démarche Qualité (Back, Front, App)
+* Méthodologie : Agilité, DevOps, TDD, BDD.
+* Next steps : Kafka, Continuous Delivery.
+
+Monter en compétences :
+* Learning permanent
+* Gitlab, merge request, code review
+* Metrics jenkins sur ce qui ne va pas / CI jenkins
+* Dashboard physique
+* Personne extérieures qui viennent présenter des choses nouvelles et enrichir l'équipe.
+
+LA phrase du jour : L'intégration continue c'est "Harder better faster stronger" (Daft Punk) :)
+
+[Superbe présentation à découvrir sur Slideshare](http://fr.slideshare.net/meeticTech/meetic-backend-mutation-with-symfony)
+
+
+## Faites plaisir à vos utilisateurs : surveillez votre prod par Grégoire Pineau
+
+Développer une application Symfony est maintenant chose commune, mais en connaissez-vous vraiment le comportement en production ? [Le SpeakerDeck Monitorer sa prod](https://speakerdeck.com/lyrixx/symfony-live-2015-paris-monitorer-sa-prod)
+
+
+## Symfony Debug et VarDumper, ou comment debugger comfortablement par Nicolas Grekas
+
+Il n'y a que des bugs faciles à résoudre ...quand on a de quoi les cerner ! [Voir la présentation Symfony Debug et VarDumper](http://fr.slideshare.net/nicolas.grekas/symfony-debug-vardumper)
```

Comment:
* space should be aftre ellipsis
* Ellipsis is a single character `…` (not three dots)